packages/tuono/src/build/config/create-json-config.ts

```diff
@@ -0,0 +1,48 @@
+import { promises as fs } from 'fs'
```

Comment:
for import consistency across the codebase

```suggestion
import fs from 'fs/promises'
```